fe1-web/src/features/popcha/network/index.ts

```diff
@@ -0,0 +1,17 @@
+import { ActionType, ObjectType, ProcessableMessage } from 'core/network/jsonrpc/messages';
+
+import { PopchaConfiguration } from '../interface';
+import { PopchaAuthMsg } from './messages/PopchaAuthMsg';
+
+export const configureNetwork = (configuration: PopchaConfiguration) => {
+  configuration.messageRegistry.add(
+    ObjectType.POPCHA,
+    ActionType.AUTH,
+    // We cannot subscribe on this channel, so we should not receive any message
+    (_: ProcessableMessage) => {
+      console.log(`Received unprocessed popcha auth message${JSON.stringify(_)}`);
+      return true;
```

Comment:
Then would you consider it as an error? If it is never supposed to happen from a security perspective, it may be better to throw an error.

In either way, you can just completely drop the "ProcessableMessage" type since you're not using it:
```suggestion
    // We cannot subscribe on this channel, so we should not receive any message
    () => {
      console.log(`Received unprocessed popcha auth message ${JSON.stringify(_)}`);
      return true;
```